### PR TITLE
Create script to add admin

### PR DESCRIPTION
**Pull request by pako535**
Creating a script that allows you to create an administrator account by running it in the console.

### CODE REVIEW

File: utils/create_admin_user.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
+from werkzeug.security import generate_password_hash
+import re
```
[the remark this answers is not in the document]
Comment (wojwarych) [reply utils/create_admin_user.py]: remove this import @Jahenstein

File: utils/create_admin_user.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+from config import Config
+from forms.custom_validators import email_regex
```
[the remark this answers is not in the document]
Comment (wojwarych) [reply utils/create_admin_user.py]: as this one too

File: utils/create_admin_user.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
+from werkzeug.security import generate_password_hash
```
[the remark this answers is not in the document]
Comment (wojwarych) [reply utils/create_admin_user.py]: @Jahenstein  try to add `# noqa` at the end of this line
As far as I remember stickler also uses flake8 or something like this
Lines with `# noqa` are ignored by linter

File: utils/create_admin_user.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Jahenstein) [reply utils/create_admin_user.py]: I am using re in line 18.

File: utils/create_admin_user.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (Jahenstein) [reply utils/create_admin_user.py]: Stickler got crazy "generate_password_hash" is used in line 26/

File: utils/create_admin_user.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (Jahenstein) [reply utils/create_admin_user.py]: Used in line 18.

**Pull request merged**